packages/flutter_tools/test/general.shard/runner/flutter_command_test.dart

```diff
@@ -496,6 +505,16 @@ void main() {
       await flutterCommand.run();
 
       expect(usage.timings, isEmpty);
+      // Iterate through and count all the [Event.timing] instances
+      int timingEventCounts = 0;
+      for (final Event e in fakeAnalytics.sentEvents) {
+        if (e.eventName == DashEvent.timing) {
+          timingEventCounts += 1;
+        }
+      }
+      expect(timingEventCounts, 0,
+          reason: 'There should not be any timing events sent, '
+              'there may be other events');
```

Comment:
```suggestion
      expect(
        timingEventCounts,
        0,
        reason: 'There should not be any timing events sent, there may be other events',
      );
```

Reply:
why would there be other events?

Reply:
oh, this test was only ever testing that there weren't timing events?

Reply:
Correct, with package:usage, we had separate lists for timing, commands, and other events, but with the new approach they are stored in the same `sentEvents` list.